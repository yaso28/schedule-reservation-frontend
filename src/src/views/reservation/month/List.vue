<template>
  <div>
    <slot name="h1" />

    <CCard v-if="loaded">
      <CCardHeader>
        <CForm @submit.prevent="search">
          <CRow
            form
            class="form-group mb-0"
          >
            <CCol
              sm="6"
              class="form-inline"
            >
              <select
                id="search-year"
                v-model="searchYear"
                class="form-control"
              >
                <option
                  v-for="year in yearList"
                  :key="year"
                  :value="year"
                  :label="year"
                />
              </select>
              <label
                for="search-year"
                class="m-2"
              >
                {{ $t('month.year') }}
              </label>
              <select
                id="search-month"
                v-model="searchMonth"
                class="form-control"
              >
                <option
                  v-for="month in monthList"
                  :key="month"
                  :value="month"
                  :label="month"
                />
              </select>
              <label
                for="search-month"
                class="m-2"
              >
                {{ $t('month.month') }}{{ $t('month.later') }}
              </label>
            </CCol>
            <CCol sm="6">
              <CButton
                type="submit"
                color="info"
              >
                {{ $t('action.search') }}
              </CButton>
            </CCol>
          </CRow>
        </CForm>
      </CCardHeader>

      <CCardBody>
        <CDataTable
          :items="itemsFiltered"
          :fields="fields"
        >
          <template #reservation_status="{item}">
            <td>
              {{ item.reservation_status.name }}
            </td>
          </template>
          <template #schedule_status="{item}">
            <td>
              {{ item.schedule_status.name }}
            </td>
          </template>
          <template #action="{item}">
            <td>
              <CLink
                class="btn btn-outline-primary"
                :to="{ name: 'reservation.schedule.list', query: { from: item.first_day, to: item.last_day } }"
              >
                {{ $t('schedule.model') }}
              </CLink>
              <CLink
                class="btn btn-outline-primary"
                :to="{ name: 'reservation.reservation.list', query: { from: item.first_day, to: item.last_day } }"
              >
                {{ $t('reservation.model') }}
              </CLink>
              <CLink
                v-if="canWrite"
                class="btn btn-outline-primary"
                :to="{ name: 'reservation.month.send', params: { id: item.id } }"
              >
                {{ $t('send.model') }}
              </CLink>
            </td>
          </template>
        </CDataTable>
      </CCardBody>
    </CCard>
  </div>
</template>

<script>
import permission from '@/consts/permission'

export default {
  name: 'MonthList',
  inheritAttrs: false,
  inject: ['apiService', 'dateService'],
  data () {
    return {
      items: null,
      yearList: null,
      searchYear: null,
      searchMonth: null,
      currentYear: null,
      currentMonth: null,
      oldestYear: null,
      oldestMonth: null
    }
  },
  computed: {
    fields () {
      return [
        { key: 'name', label: this.$t('common.name') },
        { key: 'reservation_status', label: this.$t('reservation_status.model') },
        { key: 'schedule_status', label: this.$t('schedule_status.model') },
        { key: 'action', label: '' }
      ]
    },
    monthList () {
      return Array.from(Array(12), (v, i) => i + 1)
    },
    loaded () {
      return this.items
    },
    itemsFiltered () {
      return (this.items ?? []).filter(
        item => this.currentYear < item.year || (this.currentYear === item.year && this.currentMonth <= item.month)
      )
    },
    canWrite () {
      return this.$store.getters['auth/can'][permission.reservation.write]
    }
  },
  mounted () {
    this.init()
  },
  methods: {
    init () {
      const today = this.dateService.today()
      this.searchYear = this.dateService.year(today)
      this.searchMonth = this.dateService.month(today)
      this.yearList = Array.from(Array(4), (v, i) => this.searchYear - 2 + i)
      this.search()
    },
    search () {
      this.currentYear = this.searchYear
      this.currentMonth = this.searchMonth
      if (!this.oldestYear || this.oldestYear > this.currentYear || (this.oldestYear === this.currentYear && this.oldestMonth > this.currentMonth)) {
        this.load()
      }
    },
    async load () {
      try {
        this.$store.commit('processing/beginGet')
        this.items = await this.apiService.getMonthList(
          this.currentYear,
          this.currentMonth
        )
        this.oldestYear = this.currentYear
        this.oldestMonth = this.currentMonth
      } catch (e) {
        if (!this.apiService.isHandledError(e)) {
          throw e
        }
      } finally {
        this.$store.commit('processing/endGet')
      }
    }
  }
}
</script>
