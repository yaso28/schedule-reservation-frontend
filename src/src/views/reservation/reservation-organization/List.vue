<template>
  <div>
    <slot name="h1" />

    <CCard>
      <CCardBody>
        <CDataTable
          :items="items"
          :fields="fields"
        >
          <template #action="{item}">
            <td>
              <template v-if="canWrite">
                <CLink
                  class="btn btn-outline-primary"
                  :to="{ name: 'reservation.reservation-organization.edit', params: { id: item.id } }"
                >
                  {{ $t('action.edit') }}
                </CLink>
                <CLink
                  class="btn btn-outline-primary"
                  :to="{ name: 'reservation.reservation-organization.add', query: { original_id: item.id } }"
                >
                  {{ $t('action.copy') }}
                </CLink>
              </template>
            </td>
          </template>
        </CDataTable>
      </CCardBody>
      <CCardFooter v-if="canWrite">
        <CLink
          class="btn btn-outline-primary"
          :to="{ name: 'reservation.reservation-organization.add'}"
        >
          {{ $t('action.add') }}
        </CLink>
        <CLink
          class="btn btn-outline-primary"
          :to="{ name: 'reservation.reservation-organization.reorder'}"
        >
          {{ $t('action.reorder') }}
        </CLink>
      </CCardFooter>
    </CCard>
  </div>
</template>

<script>
import permission from '@/consts/permission'

export default {
  name: 'ReservationOrganizationList',
  inheritAttrs: false,
  inject: ['apiService'],
  data () {
    return {
      items: []
    }
  },
  computed: {
    fields () {
      return [
        { key: 'name', label: this.$t('common.name') },
        { key: 'abbreviation', label: this.$t('common.abbreviation') },
        { key: 'registration_number', label: this.$t('reservation_organization.registration_number') },
        { key: 'action', label: '' }
      ]
    },
    canWrite () {
      return this.$store.getters['auth/can'][permission.reservation.write]
    }
  },
  mounted () {
    this.load()
  },
  methods: {
    async load () {
      try {
        this.$store.commit('processing/beginGet')
        this.items = await this.apiService.getReservationOrganizationList()
      } catch (e) {
        if (!this.apiService.isHandledError(e)) {
          throw e
        }
      } finally {
        this.$store.commit('processing/endGet')
      }
    }
  }
}
</script>
