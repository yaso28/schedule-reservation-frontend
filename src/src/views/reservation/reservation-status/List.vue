<template>
  <div>
    <slot name="h1" />

    <CCard>
      <CCardBody>
        <CDataTable
          :items="items"
          :fields="fields"
        >
          <template #reserved="{item}">
            <td>
              <MyBoolIcon :value="item.reserved" />
            </td>
          </template>
        </CDataTable>
      </CCardBody>
    </CCard>
  </div>
</template>

<script>
export default {
  name: 'ReservationStatusList',
  components: {
    MyBoolIcon: require('@/components/MyBoolIcon').default
  },
  inheritAttrs: false,
  inject: ['apiService'],
  data () {
    return {
      items: []
    }
  },
  computed: {
    fields () {
      return [
        { key: 'name', label: this.$t('common.name') },
        { key: 'description', label: this.$t('reservation_status.description') },
        { key: 'reserved', label: this.$t('reservation_status.reserved') }
      ]
    }
  },
  mounted () {
    this.load()
  },
  methods: {
    async load () {
      try {
        this.$store.commit('processing/beginGet')
        this.items = await this.apiService.getReservationStatusList()
      } catch (e) {
        if (!this.apiService.isHandledError(e)) {
          throw e
        }
      } finally {
        this.$store.commit('processing/endGet')
      }
    }
  }
}
</script>
