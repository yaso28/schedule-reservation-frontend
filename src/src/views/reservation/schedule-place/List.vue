<template>
  <div>
    <slot name="h1" />

    <CCard>
      <CCardBody>
        <CDataTable
          :items="items"
          :fields="fields"
        >
          <template #price_per_hour="{item}">
            <td class="text-right text-nowrap">
              {{ $n(item.price_per_hour, "currency") }}
            </td>
          </template>
          <template #action="{item}">
            <td>
              <template v-if="canWrite">
                <CLink
                  class="btn btn-outline-primary"
                  :to="{ name: 'reservation.schedule-place.edit', params: { id: item.id } }"
                >
                  {{ $t('action.edit') }}
                </CLink>
                <CLink
                  class="btn btn-outline-primary"
                  :to="{ name: 'reservation.schedule-place.add', query: { original_id: item.id } }"
                >
                  {{ $t('action.copy') }}
                </CLink>
              </template>
            </td>
          </template>
        </CDataTable>
      </CCardBody>
      <CCardFooter v-if="canWrite">
        <CLink
          class="btn btn-outline-primary"
          :to="{ name: 'reservation.schedule-place.add'}"
        >
          {{ $t('action.add') }}
        </CLink>
        <CLink
          class="btn btn-outline-primary"
          :to="{ name: 'reservation.schedule-place.reorder'}"
        >
          {{ $t('action.reorder') }}
        </CLink>
      </CCardFooter>
    </CCard>
  </div>
</template>

<script>
import permission from '@/consts/permission'

export default {
  name: 'SchedulePlaceList',
  inheritAttrs: false,
  inject: ['apiService'],
  data () {
    return {
      items: []
    }
  },
  computed: {
    fields () {
      return [
        { key: 'name', label: this.$t('common.name') },
        { key: 'abbreviation', label: this.$t('common.abbreviation') },
        { key: 'price_per_hour', label: this.$t('schedule_place.price_per_hour') },
        { key: 'action', label: '' }
      ]
    },
    canWrite () {
      return this.$store.getters['auth/can'][permission.reservation.write]
    }
  },
  mounted () {
    this.load()
  },
  methods: {
    async load () {
      try {
        this.$store.commit('processing/beginGet')
        this.items = await this.apiService.getSchedulePlaceList()
      } catch (e) {
        if (!this.apiService.isHandledError(e)) {
          throw e
        }
      } finally {
        this.$store.commit('processing/endGet')
      }
    }
  }
}
</script>
