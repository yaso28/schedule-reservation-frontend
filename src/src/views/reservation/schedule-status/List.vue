<template>
  <div>
    <slot name="h1" />

    <CCard>
      <CCardBody>
        <CDataTable
          :items="items"
          :fields="fields"
        >
          <template #display_type="{item}">
            <td>
              <CBadge
                v-if="item.display_type"
                :color="item.display_type"
              >
                {{ item.display_type }}
              </CBadge>
            </td>
          </template>
          <template #is_public="{item}">
            <td>
              <MyBoolIcon :value="item.is_public" />
            </td>
          </template>
          <template #bulk_change_mode="{item}">
            <td>
              {{ $t(`schedule_status.bulk_change_mode.${item.bulk_change_mode}`) }}
            </td>
          </template>
        </CDataTable>
      </CCardBody>
    </CCard>
  </div>
</template>

<script>
export default {
  name: 'ScheduleStatusList',
  components: {
    MyBoolIcon: require('@/components/MyBoolIcon').default
  },
  inheritAttrs: false,
  inject: ['apiService'],
  data () {
    return {
      items: []
    }
  },
  computed: {
    fields () {
      return [
        { key: 'name', label: this.$t('common.name') },
        { key: 'display_type', label: this.$t('schedule_status.display_type') },
        { key: 'is_public', label: this.$t('schedule_status.is_public') },
        { key: 'bulk_change_mode', label: this.$t('schedule_status.bulk_change_mode.model') }
      ]
    }
  },
  mounted () {
    this.load()
  },
  methods: {
    async load () {
      try {
        this.$store.commit('processing/beginGet')
        this.items = await this.apiService.getScheduleStatusList()
      } catch (e) {
        if (!this.apiService.isHandledError(e)) {
          throw e
        }
      } finally {
        this.$store.commit('processing/endGet')
      }
    }
  }
}
</script>
