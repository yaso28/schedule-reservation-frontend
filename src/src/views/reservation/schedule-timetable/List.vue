<template>
  <div>
    <slot name="h1" />

    <CCard>
      <CCardBody>
        <CDataTable
          :items="items"
          :fields="fields"
        >
          <template #action="{item}">
            <td>
              <template v-if="canWrite">
                <CLink
                  class="btn btn-outline-primary"
                  :to="{ name: 'reservation.schedule-timetable.edit', params: { id: item.id } }"
                >
                  {{ $t('action.edit') }}
                </CLink>
                <CLink
                  class="btn btn-outline-primary"
                  :to="{ name: 'reservation.schedule-timetable.add', query: { original_id: item.id } }"
                >
                  {{ $t('action.copy') }}
                </CLink>
              </template>
            </td>
          </template>
        </CDataTable>
      </CCardBody>
      <CCardFooter v-if="canWrite">
        <CLink
          class="btn btn-outline-primary"
          :to="{ name: 'reservation.schedule-timetable.add'}"
        >
          {{ $t('action.add') }}
        </CLink>
        <CLink
          class="btn btn-outline-primary"
          :to="{ name: 'reservation.schedule-timetable.reorder'}"
        >
          {{ $t('action.reorder') }}
        </CLink>
      </CCardFooter>
    </CCard>
  </div>
</template>

<script>
import permission from '@/consts/permission'

export default {
  name: 'ScheduleTimetableList',
  inheritAttrs: false,
  inject: ['apiService'],
  data () {
    return {
      items: []
    }
  },
  computed: {
    fields () {
      return [
        { key: 'name', label: this.$t('common.name') },
        { key: 'details', label: this.$t('schedule_timetable.details'), _classes: 'nl2br' },
        { key: 'action', label: '' }
      ]
    },
    canWrite () {
      return this.$store.getters['auth/can'][permission.reservation.write]
    }
  },
  mounted () {
    this.load()
  },
  methods: {
    async load () {
      try {
        this.$store.commit('processing/beginGet')
        this.items = await this.apiService.getScheduleTimetableList()
      } catch (e) {
        if (!this.apiService.isHandledError(e)) {
          throw e
        }
      } finally {
        this.$store.commit('processing/endGet')
      }
    }
  }
}
</script>
