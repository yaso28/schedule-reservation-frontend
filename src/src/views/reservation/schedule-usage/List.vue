<template>
  <div>
    <slot name="h1" />

    <CCard>
      <CCardBody>
        <CDataTable
          :items="items"
          :fields="fields"
        >
          <template #is_public="{item}">
            <td>
              <MyBoolIcon :value="item.is_public" />
            </td>
          </template>
          <template #reservation_organization="{item}">
            <td>
              {{ item.reservation_organization.name }}
            </td>
          </template>
          <template #action="{item}">
            <td>
              <template v-if="canWrite">
                <CLink
                  class="btn btn-outline-primary"
                  :to="{ name: 'reservation.schedule-usage.edit', params: { id: item.id } }"
                >
                  {{ $t('action.edit') }}
                </CLink>
                <CLink
                  class="btn btn-outline-primary"
                  :to="{ name: 'reservation.schedule-usage.add', query: { original_id: item.id } }"
                >
                  {{ $t('action.copy') }}
                </CLink>
              </template>
            </td>
          </template>
        </CDataTable>
      </CCardBody>
      <CCardFooter v-if="canWrite">
        <CLink
          class="btn btn-outline-primary"
          :to="{ name: 'reservation.schedule-usage.add'}"
        >
          {{ $t('action.add') }}
        </CLink>
        <CLink
          class="btn btn-outline-primary"
          :to="{ name: 'reservation.schedule-usage.reorder'}"
        >
          {{ $t('action.reorder') }}
        </CLink>
      </CCardFooter>
    </CCard>
  </div>
</template>

<script>
import permission from '@/consts/permission'

export default {
  name: 'ScheduleUsageList',
  components: {
    MyBoolIcon: require('@/components/MyBoolIcon').default
  },
  inheritAttrs: false,
  inject: ['apiService'],
  data () {
    return {
      items: []
    }
  },
  computed: {
    fields () {
      return [
        { key: 'name', label: this.$t('common.name') },
        { key: 'is_public', label: this.$t('schedule_usage.is_public') },
        { key: 'reservation_organization', label: this.$t('reservation_organization.model') },
        { key: 'action', label: '' }
      ]
    },
    canWrite () {
      return this.$store.getters['auth/can'][permission.reservation.write]
    }
  },
  mounted () {
    this.load()
  },
  methods: {
    async load () {
      try {
        this.$store.commit('processing/beginGet')
        this.items = await this.apiService.getScheduleUsageList()
      } catch (e) {
        if (!this.apiService.isHandledError(e)) {
          throw e
        }
      } finally {
        this.$store.commit('processing/endGet')
      }
    }
  }
}
</script>
