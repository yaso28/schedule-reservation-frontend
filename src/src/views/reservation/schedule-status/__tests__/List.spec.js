import List from '../List'
import mountOptionCreator from '@/testUtils/mountOptionCreator'
import pageTester from '@/testUtils/pageTester'
import promiseMocker from '@/testUtils/promiseMocker'
import storeModules from '@/store/modules'

jest.mock('@/store/modules')

describe('List', () => {
  let mountOption
  let apiService
  beforeEach(() => {
    promiseMocker.init()
    mountOption = mountOptionCreator.create(storeModules)
    apiService = mountOption.provide.apiService
  })

  afterEach(() => {
    promiseMocker.clear()
    jest.resetAllMocks()
  })

  describe('マウント', () => {
    let wrapper
    let loadSwitch
    let loadResult
    let loadError
    beforeEach(async () => {
      loadSwitch = promiseMocker.createSwitch()
      loadResult = require('@/testUtils/dummyApiData/schedule-status/list.json')
      loadError = new Error('load')
      apiService.getScheduleStatusList = promiseMocker.mock(loadResult, loadError, loadSwitch)
      wrapper = await pageTester.mount(List, { name: 'reservation.schedule-status.list' }, mountOption)
    })

    test('titleタグ', () => {
      pageTester.assertTitleTag()
    })

    test('metaタグ', () => {
      pageTester.assertMetaTags()
    })

    test('ページ全体表示', () => {
      expect(wrapper.element).toMatchSnapshot()
    })

    test('Processingカバー表示開始', () => {
      expect(storeModules.processing.mutations.beginGet).toHaveBeenCalled()
    })

    test('データ取得処理を呼出', () => {
      expect(apiService.getScheduleStatusList).toHaveBeenCalledWith()
    })

    test('Processingカバー表示中', () => {
      expect(
        storeModules.processing.mutations.endGet
      ).not.toHaveBeenCalled()
    })

    describe('データ取得：エラー', () => {
      beforeEach(() => {
        promiseMocker.setSwitch(loadSwitch, false)
        apiService.isHandledError = jest.fn().mockReturnValueOnce(true)
        promiseMocker.flushPendings()
      })

      test('エラー処理', () => {
        expect(apiService.isHandledError).toHaveBeenCalledWith(loadError)
      })

      test('Processingカバー表示終了', () => {
        expect(storeModules.processing.mutations.endGet).toHaveBeenCalled()
      })
    })

    describe('データ取得：成功', () => {
      beforeEach(() => {
        promiseMocker.setSwitch(loadSwitch, true)
        promiseMocker.flushPendings()
      })

      test('テーブル表示', () => {
        expect(wrapper.find('table').element).toMatchSnapshot()
      })

      test('Processingカバー表示終了', () => {
        expect(storeModules.processing.mutations.endGet).toHaveBeenCalled()
      })
    })
  })
})
