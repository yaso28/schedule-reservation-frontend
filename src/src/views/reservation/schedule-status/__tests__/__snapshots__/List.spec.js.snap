// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List マウント metaタグ 1`] = `
NodeList [
  <meta
    content="noindex"
    data-vmid="robots"
    data-vue-meta="2"
    name="robots"
  />,
]
`;

exports[`List マウント titleタグ 1`] = `"やそスポーツクラブ | 公開ステータス一覧"`;

exports[`List マウント データ取得：成功 テーブル表示 1`] = `
<table
  class="table"
  data-v-5f7284b9=""
>
  <thead
    data-v-5f7284b9=""
  >
    <tr
      data-v-5f7284b9=""
    >
      <th
        class=""
        data-v-5f7284b9=""
        style="vertical-align: middle; overflow: hidden;"
      >
        <div
          data-v-5f7284b9=""
        >
          名称
        </div>
        <!---->
      </th>
      <th
        class=""
        data-v-5f7284b9=""
        style="vertical-align: middle; overflow: hidden;"
      >
        <div
          data-v-5f7284b9=""
        >
          表示種別
        </div>
        <!---->
      </th>
      <th
        class=""
        data-v-5f7284b9=""
        style="vertical-align: middle; overflow: hidden;"
      >
        <div
          data-v-5f7284b9=""
        >
          公開フラグ
        </div>
        <!---->
      </th>
      <th
        class=""
        data-v-5f7284b9=""
        style="vertical-align: middle; overflow: hidden;"
      >
        <div
          data-v-5f7284b9=""
        >
          一括確定種別
        </div>
        <!---->
      </th>
    </tr>
    <!---->
  </thead>
  <tbody
    class="position-relative"
    data-v-5f7284b9=""
  >
    <tr
      data-v-5f7284b9=""
    >
      <td
        class=""
        data-v-5f7284b9=""
      >
         未定 
      </td>
      <td
        data-v-5f7284b9=""
      >
        <span
          class="badge badge-warning"
          data-v-5f7284b9=""
        >
          
              warning
            
        </span>
      </td>
      <td
        data-v-5f7284b9=""
      >
        <svg
          class="c-icon"
          data-v-5f7284b9=""
          role="img"
          viewBox="0 0 512 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <polygon
            class="ci-primary"
            fill="var(--ci-primary-color, currentColor)"
            points="200.359 382.269 61.057 251.673 82.943 228.327 199.641 337.731 428.686 108.687 451.314 131.313 200.359 382.269"
          />
        </svg>
      </td>
      <td
        data-v-5f7284b9=""
      >
        
            対象
          
      </td>
    </tr>
    <!---->
    <tr
      data-v-5f7284b9=""
    >
      <td
        class=""
        data-v-5f7284b9=""
      >
         確定 
      </td>
      <td
        data-v-5f7284b9=""
      >
        <!---->
      </td>
      <td
        data-v-5f7284b9=""
      >
        <svg
          class="c-icon"
          data-v-5f7284b9=""
          role="img"
          viewBox="0 0 512 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <polygon
            class="ci-primary"
            fill="var(--ci-primary-color, currentColor)"
            points="200.359 382.269 61.057 251.673 82.943 228.327 199.641 337.731 428.686 108.687 451.314 131.313 200.359 382.269"
          />
        </svg>
      </td>
      <td
        data-v-5f7284b9=""
      >
        
            変更先
          
      </td>
    </tr>
    <!---->
    <tr
      data-v-5f7284b9=""
    >
      <td
        class=""
        data-v-5f7284b9=""
      >
         中止 
      </td>
      <td
        data-v-5f7284b9=""
      >
        <span
          class="badge badge-danger"
          data-v-5f7284b9=""
        >
          
              danger
            
        </span>
      </td>
      <td
        data-v-5f7284b9=""
      >
        <svg
          class="c-icon"
          data-v-5f7284b9=""
          role="img"
          viewBox="0 0 512 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <polygon
            class="ci-primary"
            fill="var(--ci-primary-color, currentColor)"
            points="200.359 382.269 61.057 251.673 82.943 228.327 199.641 337.731 428.686 108.687 451.314 131.313 200.359 382.269"
          />
        </svg>
      </td>
      <td
        data-v-5f7284b9=""
      >
        
            対象外
          
      </td>
    </tr>
    <!---->
    <tr
      data-v-5f7284b9=""
    >
      <td
        class=""
        data-v-5f7284b9=""
      >
         非公開 
      </td>
      <td
        data-v-5f7284b9=""
      >
        <!---->
      </td>
      <td
        data-v-5f7284b9=""
      >
        <svg
          class="c-icon"
          data-v-5f7284b9=""
          role="img"
          viewBox="0 0 512 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <polygon
            class="ci-primary"
            fill="var(--ci-primary-color, currentColor)"
            points="427.314 107.313 404.686 84.687 256 233.373 107.314 84.687 84.686 107.313 233.373 256 84.686 404.687 107.314 427.313 256 278.627 404.686 427.313 427.314 404.687 278.627 256 427.314 107.313"
          />
        </svg>
      </td>
      <td
        data-v-5f7284b9=""
      >
        
            対象外
          
      </td>
    </tr>
    <!---->
    <!---->
  </tbody>
  <!---->
</table>
`;

exports[`List マウント ページ全体表示 1`] = `
<div>
  <h1>
    公開ステータス一覧
  </h1>
   
  <div
    class="card"
  >
    <div
      class="card-body"
    >
      <div
        data-v-5f7284b9=""
      >
        <!---->
        <div
          class="position-relative table-responsive"
          data-v-5f7284b9=""
        >
          <table
            class="table"
            data-v-5f7284b9=""
          >
            <thead
              data-v-5f7284b9=""
            >
              <tr
                data-v-5f7284b9=""
              >
                <th
                  class=""
                  data-v-5f7284b9=""
                  style="vertical-align: middle; overflow: hidden;"
                >
                  <div
                    data-v-5f7284b9=""
                  >
                    名称
                  </div>
                  <!---->
                </th>
                <th
                  class=""
                  data-v-5f7284b9=""
                  style="vertical-align: middle; overflow: hidden;"
                >
                  <div
                    data-v-5f7284b9=""
                  >
                    表示種別
                  </div>
                  <!---->
                </th>
                <th
                  class=""
                  data-v-5f7284b9=""
                  style="vertical-align: middle; overflow: hidden;"
                >
                  <div
                    data-v-5f7284b9=""
                  >
                    公開フラグ
                  </div>
                  <!---->
                </th>
                <th
                  class=""
                  data-v-5f7284b9=""
                  style="vertical-align: middle; overflow: hidden;"
                >
                  <div
                    data-v-5f7284b9=""
                  >
                    一括確定種別
                  </div>
                  <!---->
                </th>
              </tr>
              <!---->
            </thead>
            <tbody
              class="position-relative"
              data-v-5f7284b9=""
            >
              <tr
                data-v-5f7284b9=""
              >
                <td
                  colspan="4"
                  data-v-5f7284b9=""
                >
                  <div
                    class="text-center my-5"
                    data-v-5f7284b9=""
                  >
                    <h2
                      data-v-5f7284b9=""
                    >
                       No items 
                      <svg
                        class="text-danger mb-2 c-icon c-icon-custom-size"
                        data-v-5f7284b9=""
                        role="img"
                        viewBox="0 0 512 512"
                        width="30"
                        xmlns="http://www.w3.org/2000/svg"
                      >
                        <path
                          class="ci-primary"
                          d="M425.706,86.294A240,240,0,0,0,86.294,425.705,240,240,0,0,0,425.706,86.294ZM256,48A207.1,207.1,0,0,1,391.528,98.345L98.345,391.528A207.1,207.1,0,0,1,48,256C48,141.309,141.309,48,256,48Zm0,416a207.084,207.084,0,0,1-134.986-49.887l293.1-293.1A207.084,207.084,0,0,1,464,256C464,370.691,370.691,464,256,464Z"
                          fill="var(--ci-primary-color, currentColor)"
                        />
                      </svg>
                    </h2>
                  </div>
                </td>
              </tr>
            </tbody>
            <!---->
          </table>
          <!---->
        </div>
        <!---->
      </div>
    </div>
  </div>
</div>
`;
